### PR TITLE
Unit test for learning UDTFs

### PR DESCRIPTION
**Pull request by Lewuathe**
Two points to be discussed

* Using jitpack (HiveRunner is not published on maven central)
* Necessary to do versionup of hadoop and hive

### CODE REVIEW

**Comment by Lewuathe** [comment]
Hmm. It seems fail only on OpenJDK7.

**Comment by maropu** [comment]
@Lewuathe This always happens? Or, it's just flaky?

**Comment by Lewuathe** [comment]
@maropu This problem is always occurred only on TravisCI OpenJDK7 environment.
I have asked TravisCI support several times, but in vain...
https://twitter.com/Lewuathe/status/669743332680069120

OpenJDK7 itself seems to be fine on my VirutalBox ubuntu.

File: core/pom.xml [hunk 1]
```diff
@@ -13,18 +13,29 @@
...
+
 	<dependencies>
 		<!-- provided scope -->
 		<dependency>
 			<groupId>org.apache.hadoop</groupId>
-			<artifactId>hadoop-core</artifactId>
-			<version>0.20.2-cdh3u6</version>
-			<scope>provided</scope>
+			<artifactId>hadoop-common</artifactId>
+			<version>2.6.2</version>
```
Comment (myui) [inline core/pom.xml]: Hivemall assume to use hadoop 2.4.0 for it's dependecy.

File: core/pom.xml [hunk 2]
```diff
@@ -13,18 +13,29 @@
...
+		<dependency>
+			<groupId>org.apache.hadoop</groupId>
+			<artifactId>hadoop-mapreduce-client-core</artifactId>
+			<version>2.6.2</version>
 		</dependency>
 		<dependency>
 			<groupId>org.apache.hive</groupId>
 			<artifactId>hive-exec</artifactId>
-			<version>0.11.0</version>
+			<version>0.14.0</version>
```
Comment (myui) [inline core/pom.xml]: Hivemall assume to use Hive 0.11.0 for it's dependency.

**Comment by Lewuathe** [comment]
See #336

**Comment by myui** [comment]
close this ticket and continue discussion in #336

**Pull request closed without merging**